src/cbmc/bmc.h

```diff
@@ -287,7 +299,10 @@ class path_explorert:public bmct
   " --unwinding-assertions       generate unwinding assertions\n" \
   " --partial-loops              permit paths with partial loops\n" \
   " --no-pretty-names            do not simplify identifiers\n" \
+  " --goto-priority <jump|next>  jump: execute the target of a goto first.\n" \
+  "                              next: execute the instruction following\n" \
+  "                              the goto first. Default: 'next', unless\n" \
+  "                              --paths is passed.\n" \
```

Comment:
I think it would be better to have a more general --path-strategy or --path-selection-heuristic or something similar because this gives two strategies but it would be nice to have the option able to support others.  Depth-first, unexplored-first, etc.